recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "1.1.1" %}
+{% set version = "1.1.2" %}
 
 package:
   name: responder
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/r/responder/responder-{{ version }}.tar.gz
-  sha256: c2232012ce346104a4f0e225731cab24bf80ba460b8636d1a6c05343d1c95b52
+  sha256: a02d4329d691e0d8d4ec1a9a7d7857b93ae9c835fd56faf550ec0ee5df72d950
 
 build:
-  number: 1002
+  number: 1000
```

Comment:
Is this correct?
i.e. is it valid to drop the build number back down to 1000 when bumping the version?

Reply:
Well, the 🤖 did it, he's the boss...😉